src/ArduinoJson/MsgPack/MsgPackSerializer.hpp

```diff
@@ -123,6 +123,21 @@ class MsgPackSerializer : public VariantDataVisitor<size_t> {
     return bytesWritten();
   }
 
+  size_t visit(MsgPackBinary value) {
+    if (value.size() <= 0xFF) {
+      writeByte(0xC4);
+      writeInteger(uint8_t(value.size()));
+    } else if (value.size() <= 0xFFFF) {
+      writeByte(0xC5);
+      writeInteger(uint16_t(value.size()));
+    } else {
+      writeByte(0xC6);
```

Comment:
This line is not covered by the tests.
Please add a test in `string_length_size_4.cpp`